Files/LUISApp.json

```diff
@@ -162,7 +162,7 @@
       "entities": []
     },
     {
-      "text": "list idea in progress",
```

Comment:
This is not a typo. It was specially created for better compatibility in case that users may misspell it.
You can find "list ideas in progress" on [line 160](https://github.com/OfficeDev/Office-365-Huddle-Templates/blob/master/Files/LUISApp.json#L160)

Reply:
Rolled back